samples/sql-persistence/simple/SqlPersistence_3/EndpointMySql/EndpointMySql.Core.csproj

```diff
@@ -14,7 +14,7 @@
     <PackageReference Include="MySql.Data" Version="6.*" />
     <PackageReference Include="Newtonsoft.Json" Version="10.*" />
     <PackageReference Include="NServiceBus" Version="7.0.0-*" />
-    <PackageReference Include="NServiceBus.Persistence.Sql" Version="3.0.0-*" />
+    <PackageReference Include="NServiceBus.Persistence.Sql" Version="3.1.0-pullrequest0183*" />
```

Comment:
Seems some files were left over? We should also remove all *.Core.csproj files, correct?